esmvaltool/config-developer.yml

```diff
@@ -25,6 +25,12 @@ CMIP6:
     DKRZ: '[institute]/[dataset]/[exp]/[ensemble]/[mip]/[short_name]/[grid]/[latestversion]'
     ETHZ: '[exp]/[mip]/[short_name]/[dataset]/[ensemble]/[grid]/'
   input_file: '[short_name]_[mip]_[dataset]_[exp]_[ensemble]_[grid]_*.nc'
+  input_fx_dir:
+    default: '/'
+    BADC: '[institute]/[dataset]/[exp]/[ensemble]/[mip]/[short_name]/[grid]/[latestversion]'
+  input_fx_file: '[short_name]_[mip]_[dataset]_[exp]_[ensemble]_[grid].nc'
```

Comment:
Do we really need `input_fx_dir` and `input_fx_file`? It seems to be covered by `input_dir` and `input_file`. The only necessary change is to remove the extra `_` in front of the `*` in `input_file`.

Reply:
yes, exactly, but will keep this until we remove (if we do that) the extra string template for regular variable files; note that the file finder function will still need to be a special one since mip for CMIP5 and CMIP6 needs to be handled separately for fx variables

Reply:
> yes, exactly, but will keep this until we remove (if we do that) the extra string template for regular variable files

I don't know what you mean with "the extra string template for regular variable files". Is it just `input_file`?

> note that the file finder function will still need to be a special one since mip for CMIP5 and CMIP6 needs to be handled separately for fx variables

Hm. Just looking at `config-developer.yml`, this does *not* seem to be the case: Also in CMIP5 these variables have `[mip]` (`fx`), `[frequency]`(`fx`), and `[ensemble]` (`r0i0p0`), bringing the templates in exact agreement with those for all the other normal variables.

Reply:
have a look at `_config,replace_mip_fx(fx_file)` and `_config._get_cmip6_fx_mip(fx_var_name)` - there is just not enough information encoded in the `input_file` and `input_dir` from config-developer to be able to retrieve the correct fx files using just those, at the moment

Reply:
Maybe I don't understand, but if you look at the lines above they are literally identical, so how does this add any information?